openreview/venue/process/custom_stage_process.py

```diff
@@ -113,9 +117,11 @@ def process(client, edit, invitation):
     reviewers_name = domain.get_content_value('reviewers_name')
     reviewers_submitted_name = domain.get_content_value('reviewers_submitted_name')
     paper_reviewers_id = f'{paper_group_id}/{reviewers_name}'
+    paper_reviewers_group = openreview.tools.get_group(client, paper_reviewers_id)
     reviewers_id = domain.get_content_value('reviewers_id')
     paper_reviewers_submitted_id = f'{paper_group_id}/{reviewers_submitted_name}'
-    if 'everyone' in note.readers or reviewers_id in note.readers or paper_reviewers_id in note.readers:
+    paper_reviewers_submitted_group = openreview.tools.get_group(client, paper_reviewers_submitted_id)
+    if 'everyone' in note.readers or reviewers_id in note.readers or (paper_reviewers_id in note.readers and paper_reviewers_group):
```

Comment:
Could it happen that the `paper_reviewers_group` doesn't exist and note.readers == ['everyone']? This would still give an error, right?

Reply:
Oh yeah I'll just move that check further out